### PR TITLE
Separate initilizers from inputs in target opset 9+

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

File: keras2onnx/topology.py [hunk 1]
```diff
@@ -233,27 +233,18 @@ def convert_topology(topology, model_name, doc_string, target_opset, channel_fir
         keras2onnx_logger().debug("Converting the operator (%s): %s" % (operator.full_name, operator.type))
         get_converter(operator.type)(scope, operator, container)
 
-    # When calling ModelComponentContainer's add_initializer(...), nothing is added into the input list. However, in
```
Comment (wenbingl) [inline keras2onnx/topology.py]: where is this code block for opset < 9?

File: keras2onnx/topology.py [hunk 2]
```diff
@@ -233,27 +233,34 @@ def convert_topology(topology, model_name, doc_string, target_opset, channel_fir
...
+            value_info = helper.make_tensor_value_info(tensor.name, tensor.data_type, tensor.dims)
+            extra_inputs.append(value_info)
 
     # enable the ONNX optimizations
     try:
         import onnxtk
-        nodes = onnxtk.optimizer.optimize_onnx(container.nodes, nchw_inputs=nchw_inputs, inputs=container.inputs + extra_inputs,
-                              outputs=container.outputs)
+        if target_opset < 9:
+            nodes = onnxtk.optimizer.optimize_onnx(container.nodes, nchw_inputs=nchw_inputs,
```
Comment (wenbingl) [inline keras2onnx/topology.py]: where is onnxtk change?

Comment (jiafatom) [reply keras2onnx/topology.py]: https://github.com/onnx/onnxmltools/pull/263

**Comment by jiafatom** [comment]
No, we want the final container.initializers separate from container.input. It is fine to put initializers into extra_inputs for optimizer.

**Pull request merged**